lib/annotate/annotate_models.rb

```diff
@@ -908,6 +913,58 @@ def get_attributes(column, column_type, klass, options)
 
       attrs
     end
+
+    # Returns the regular expression for finding a column definition based on
+    # the format of the annotations being made
+    #
+    # == Returns:
+    # Regular expression
+    #
+    # @param [Symbol] symbol representation of the format being used for
+    #   annotations
+    def column_pattern_for(format_type)
+      case format_type
+      when :markdown then /^#\s+\*{2}`\w+`\*{2}(?:\s+\|\s`.*`)*$/
+      when :rdoc then
+        field_name_regex = /^#\s+\*\w+\*\:{2}/
+        field_info_regex = /\s+\<tt\>.*\<\/tt\>$/
```

Comment:
Similar to above, rather than checking the `<tt>...</tt>` this could all probably be `.*$`